app/views/promotion/weeklyLandingPage.scala.html

```diff
@@ -2,94 +2,133 @@
 @import com.gu.memsub.promo.Promotion.PromoWithWeeklyLandingPage
 @import com.gu.memsub.promo.Promotion.asAnyPromotion
 @import views.support.LandingPageOps._
+@import configuration.Links
+@import views.support.CommaSplit
 @import views.support.{MarkdownRenderer}
 @import com.gu.memsub.subsv2.Catalog
 @import model.Subscriptions._
+@import controllers.CachedAssets.hashedPathFor
 @import com.gu.i18n.Country
 @import views.support.WeeklyPromotion
-@(country: Country, catalog: Catalog, promoCode: PromoCode, promotion: PromoWithWeeklyLandingPage, md: MarkdownRenderer)
-    @voucherPlans = @{catalog.voucher.list.filter(p => promotion.appliesTo.productRatePlanIds.contains(p.id))}
-    @deliveryPlans = @{catalog.delivery.list.filter(p => promotion.appliesTo.productRatePlanIds.contains(p.id))}
-    @anyPromotion = @{asAnyPromotion(promotion)}
-    @title = @{"Subscribe to the Guardian Weekly"}
-
+@import com.gu.memsub.promo.Grey
+@import com.gu.memsub.promo.Blue
+@import com.gu.memsub.promo.White
+@(country: Country, catalog: Catalog, promoCode: Option[PromoCode], promotion: Option[PromoWithWeeklyLandingPage],description:Html, md: MarkdownRenderer)
+    @anyPromotion = @{promotion.map(asAnyPromotion)}
+    @nonTrackingPromotion = @{anyPromotion.flatMap(p => p.asDiscount orElse p.asFreeTrial orElse p.asIncentive)}
+    @title = @{"The Guardian Weekly"}
     @discountedRegions = @{WeeklyPromotion.validRegionsForPromotion(promotion, promoCode, country)(catalog)}
+    @main(title = s"$title | The Guardian", bodyClasses = List("is-wide")) {
+    @promotion.flatMap(_.landingPage.image).fold{
+        @*
+        don't ship this, use a grid image
```

Comment:
If you are actually going to ship this then either remove the comment or put a Trello ticket to tidy it up in a future PR ;)